mindsdb/integrations/handlers/gcs_handler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+duckdb
+fsspec
+gcsfs
```

Comment:
I don't see this package utilized anywhere in the code. Is it required?

Reply:
Yes, it will be used by fsspec.

Reply:
In that case, you will need to create another variable here similar to [this](https://github.com/mindsdb/mindsdb/blob/da0e83985974d7796897c25b3c40ecf57c17e485/tests/scripts/check_requirements.py#L64) and store the name of the package in there. After that, you will need to add it to this [list](https://github.com/mindsdb/mindsdb/blob/da0e83985974d7796897c25b3c40ecf57c17e485/tests/scripts/check_requirements.py#L67).